components/kyma-environment-broker/internal/error/errors.go

```diff
@@ -92,7 +92,7 @@ func ReasonForError(err error) LastError {
 		return LastError{}
 	}
 
-	cause := errors.Cause(err)
+	cause := UnwrapAll(err)
```

Comment:
@vvxxvvxx, it has been tested and this should for metric and alerting purposes be equivalent to the old error wrapping, but please keep an eye on any regressions and ping @kyma-project/gopher immediately if anything breaks.